code/modules/mob/living/carbon/carbon_defines.dm

```diff
@@ -23,4 +23,6 @@
 	var/halloss = 0		//Hallucination damage. 'Fake' damage obtained through hallucinating or the holodeck. Sleeping should cause it to wear off.
 
 	var/traumatic_shock = 0
-	var/shock_stage = 0
\ No newline at end of file
+	var/shock_stage = 0
+
+	var/no_acidprocess_flag = 0 //While this is true, the human is unaffected by acid damage from acid puddles; normally applied on being hit by acid spit/spray; reduced at a rate of 1/tick. Used to avoid double whammies.
```

Comment:
Just name this `no_acidprocess`, since it's not a real flag.

Reply:
`acid_proof` or some positive name, please. Let's avoid having a `if(!no_acidprocess)` check later on.

Reply:
`acid_proof` is misleading anyhow. This is more of a cooldown for the boiler acid spray on the mob rather than real acid proofness.